src/settings/SettingsView.ts

```diff
@@ -103,6 +103,8 @@ export class SettingsView extends BaseTopLevelView implements TopLevelView<Setti
 	private _selectedFolder: SettingsFolder<unknown>
 	private _currentViewer: UpdatableSettingsViewer | null = null
 	private showBusinessSettings: stream<boolean> = stream(false)
+	private readonly _targetFolder: string
```

Comment:
just FYI we don't need underscores in field names, that's pre-typescript convention